src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import store from './configureStore';
+
+import Greeting from './components/Greeting';
+
+const App = () => (
+  <Provider store={store}>
+    <div>
+      <Greeting />
+    </div>
+  </Provider>
+
+);
```

Comment:
- [ ] Within the App component, you are required to add the routes leading to the Greeting component with the help of `react-router` package. Add the necessary dependency and use the available methods from `react-router` to set up the route for Greeting.